emperor/support_files/js/sceneplotview3d.js

```diff
@@ -100,6 +103,26 @@ define([
     this.dimensionRanges = {'max': [], 'min': []};
     this.drawAxesWithColor(0xFFFFFF);
     this.drawAxesLabelsWithColor(0xFFFFFF);
+
+    this.raycaster = new THREE.Raycaster();
```

Comment:
This is taken care of by the line `scope.raycaster.setFromCamera(scope.mouse, scope.camera);`